app/renderer/js/pages/preference/base-section.js

```diff
@@ -16,6 +16,18 @@ class BaseSection extends BaseComponent {
 		$optionControl.addEventListener('click', clickHandler);
 	}
 
+	generateSettingDropdown(props) {
+		const {$element, value, options, clickHandler} = props;
```

Comment:
Nit: add spaces around braces.

Reply:
This was a pre-existing thing - maybe we should update our style check rules to disallow no spaces around braces?

Reply:
@kanishk98 we used to have it I think.